kernel/init.c

```diff
@@ -363,10 +363,10 @@ void z_early_boot_rand_get(uint8_t *buf, size_t length)
 {
 	int n = sizeof(uint32_t);
 #ifdef CONFIG_ENTROPY_HAS_DRIVER
-	const struct device *entropy = device_get_binding(DT_CHOSEN_ZEPHYR_ENTROPY_LABEL);
+	const struct device *entropy = DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
```

Comment:
I think you mean `DEVICE_DT_GET_OR(DT_CHOSEN(zephyr_entropy), NULL);`, right?

Reply:
fixed